Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
@andy-armstrong  I didn't see your most recent comment before.  Doesn't the cookie-cutter repo assume that it's a django app?  If that's the case, for things like this and the UI toolkit, they wouldn't be using the same base jshintrc file.  Would it make sense to create another edx-lint style repo, but for javascript quality?